Sources/BuilderIO/Components/BuilderBlock.swift

```diff
@@ -57,35 +57,65 @@ struct BuilderBlockLayout<Content: View>: View {
     let minHeight = extractPixels(responsiveStyles["minHeight"])
     let maxHeight = extractPixels(responsiveStyles["maxHeight"])
     let minWidth = extractPixels(responsiveStyles["minWidth"])
```

Comment:
Unless this was intentional, similar to maxWidth, we can add it here itself.
```suggestion
    let minWidth = extractPixels(responsiveStyles["minWidth"]) ?? 0
```